src/view/renderer.js

```diff
@@ -407,6 +406,12 @@ export default class Renderer {
 	 */
 	_updateText( viewText, options ) {
 		const domText = this.domConverter.findCorrespondingDomText( viewText );
+
+		// If this is a new text node and it is not in DOM, it will be created and handled in `_updateChildren`.
+		if ( !domText ) {
```

Comment:
I don't understand. Before there was a condition in `render` method` in the loop where `_updateText` is called. I removed this condition because I wanted `_updateText` to be fired more often. But this caused a situation where `_upadateText` was fired on text nodes that weren't yet converted to DOM. That's why I needed to add the statement in `_updateText`.